### DISCUSSION TITLE
Reset guest mode?

### DISCUSSION THREAD

**Discussion by SkullGunner** [OnionUI/Onion#565] [Q&A] [answered]
I was letting a friend really play around in guest mode and I thought once I turned guest mode on and off the settings would reset but it acts more like a second account keeping saves and settings.
Is there a way to reset the guest mode to default blank settings?

**Comment by Aemiii91** [comment, marked as answer]
With guest mode active, you can reset settings in `Apps` -> `Tweaks` -> `Advanced` -> `Reset settings...`

For saves, you can just delete the `saves` and `states` folders in the current profile directory under `Saves`.
